requirements.txt

```diff
@@ -14,3 +14,4 @@ sklearn
 sympy
 tensorflow
 xgboost
+colorama
```

Comment:
This list is sorted alphabetically so that it is easy to spot missing dependencies and duplicate entries.